model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -276,6 +267,14 @@ async def create_model_bundle(
     ) -> ModelBundle:
         if source == LLMSource.HUGGING_FACE:
             if framework == LLMInferenceFramework.DEEPSPEED:
+                if not self.docker_repository.image_exists(
```

Comment:
nit: i think we could extract a function

Reply:
where would you want the function to reside? I feel like it'd be a small one so not sure if it should be directly in the `CreateLLMModelEndpointV1UseCase` class

Reply:
I think putting the helper in this class is fine. Has less to do with the size of the function, as to its scope and relevance - IMO better to keep helpers "local" as much as possible unless they're needed outside of the module.

Reply:
Gotcha sounds good, will refactor!